provider/cmd/run.go

```diff
@@ -38,7 +39,14 @@ const (
 	// FlagK8sManifestNS
 	FlagK8sManifestNS = "k8s-manifest-ns"
 	// FlagGatewayListenAddress determines listening address for Manifests
-	FlagGatewayListenAddress = "gateway-listen-address"
+	FlagGatewayListenAddress       = "gateway-listen-address"
+	FlagBidPricingStrategy         = "bid-price-strategy"
+	FlagBidPriceCPUScale           = "bid-price-cpu-scale"
+	FlagBidPriceMemoryScale        = "bid-price-memory-scale"
+	FlagBidPriceStorageScale       = "bid-price-storage-scale"
+	FlagBidPriceScriptPath         = "bid-price-script-path"
+	FlagBidPriceScriptProcessLimit = "bid-price-script-process-limit"
+	FlagBidPriceScriptTimeout      = "bid-price-script-process-timeout"
```

Comment:
Can you add all of these to the kustomize [deployment environment](https://github.com/ovrclk/akash/blob/494572ac32a7a48d801a27397e6d2047d80419ec/_docs/kustomize/akash-provider/deployment.yaml#L212-L224) and [config](https://github.com/ovrclk/akash/blob/master/_docs/kustomize/akash-provider/kustomization.yaml#L50-L51)?